components/index.ts

```diff
@@ -505,6 +505,7 @@ function startServer(options: { hmr: boolean; pluginDevHost: boolean }): void {
         "contractSessions",
         "userdata",
         "contracts",
+        "plugins",
```

Comment:
This is actually a bug-fix for the hot-reload. I will remove those when I squash after the second round of checks.